apps/web/src/utils/page-visibility.ts

```diff
@@ -17,37 +17,50 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-var hidden, visibilityChange;
-if (typeof document.hidden !== "undefined") {
-  // Opera 12.10 and Firefox 18 and later support
-  hidden = "hidden";
-  visibilityChange = "visibilitychange";
-} else if (typeof document.msHidden !== "undefined") {
-  hidden = "msHidden";
-  visibilityChange = "msvisibilitychange";
-} else if (typeof document.webkitHidden !== "undefined") {
-  hidden = "webkitHidden";
-  visibilityChange = "webkitvisibilitychange";
-}
+const getVisibilityChangeParams = () => {
+  let hidden: "hidden" | "msHidden" | "webkitHidden" = "hidden";
+  let visibilityChange:
+    | "visibilitychange"
+    | "msvisibilitychange"
+    | "webkitvisibilitychange" = "visibilitychange";
```

Comment:
No need for statically typing these. It'll be automatically inferred by Typescript. That's why we are using a function.

Reply:
Ok